sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/AnalyzeConversationOptions.cs

```diff
@@ -8,7 +8,16 @@ namespace Azure.AI.Language.Conversations
 {
     /// <summary> The request body. </summary>
     [CodeGenModel("ConversationAnalysisOptions")]
+    [CodeGenSuppress("AnalyzeConversationOptions", typeof(string))]
     public partial class AnalyzeConversationOptions
     {
+        /// <summary> Initializes a new instance of AnalyzeConversationOptions. </summary>
+        public AnalyzeConversationOptions()
+        {
+            Parameters = new ChangeTrackingDictionary<string, AnalysisParameters>();
+        }
+
+        /// <summary> The conversation utterance to be analyzed. </summary>
+        internal string Query { get; set; }
```

Comment:
Nit: I'd use `[CodeGenMember("Query")]` to rename this to `Utterance` just for consistency throughout. IMO, the uniformity would make the code easier to maintain over time.

Reply:
Makes sense! I thought about doing this but I remembered being told to avoid internal changes (in TA) since they may take more effort to maintain, but in this case, I definitely agree.